### DISCUSSION TITLE
Wiimote Support in 0.5 official release?

### DISCUSSION THREAD

**Discussion by NeilAgar** [ndeadly/MissionControl#293] [Q&A] [answered]
I was just wondering if Nintendo controllers (like the wii(u) motes/pro controllers) would be added in the next official release. Thanks!

**Comment by ndeadly** [comment, marked as answer]
Yes, it's the sole thing holding up the final 0.5.0 release. If you search the issue tracker there are several issues where I have explained the situation already.

**Reply by NeilAgar** [reply]
Ah, I see that now, can't wait for the release!
